feature-libs/cart/wish-list/root/components/add-to-wishlist/add-to-wish-list.component.ts

```diff
@@ -59,4 +61,42 @@ export class AddToWishListComponent {
       product.stock && product.stock.stockLevelStatus !== 'outOfStock'
     );
   }
+
+  protected getWishListLoading(): Observable<boolean> {
```

Comment:
no need add these functions, just change the component's variable `wishListEntries$` and `loading$` to `get` function. something like this:
```
get wishListEntries$(): Observable<OrderEntry[]> {
    return this.wishListFacade.getWishList().pipe(
      filter((wishlist) => Boolean(wishlist)),
      map((wishList) => wishList.entries ?? [])
    );
  }
```
```
get loading$(): Observable<boolean> {
    return this.wishListFacade.getWishListLoading();
  }
```